frontend/src/components/managePermissions/table/ApplicationAdminTable.vue

```diff
@@ -152,6 +142,18 @@ const highlightNewAppAdminAccesRow = (rowData: any) => {
                     header="Domain"
                     sortable
                 ></Column>
+                <Column field="user.first_name" header="Full Name" sortable>
```

Comment:
Seems like we can only pass one field data, so the sorting will be based on the first name, which I feel is fine. Previously the design will show both first_name and last_name column, but the new design we saw will just be full name.

Reply:
I am curious (I may not be familiar with detail of this dataTable) that what this "**field**" actually do.
Can it be pass with full name as `:field="getUserFullName"`, or simply it is useless like this:
![image](https://github.com/user-attachments/assets/273cf6e9-9fdf-4113-bd6a-d486b72f908c)

Reply:
I think it links to the data we pass to the user table component, so it will be the fields inside the "userRoleAssingment" data
<img width="418" alt="Screen Shot 2024-07-23 at 2 01 36 PM" src="https://github.com/user-attachments/assets/e4758df5-9475-4ea0-b8e1-effc14696aea">

Reply:
I did try both for what I was curious at, before I asked, and the datatable sorting and filtering seems still working fine.  So I was wondering maybe it is not necessary in our datatable.
But you can leave it there if you know it does affect something.

Reply:
oh really? I know the filtering should work. This field might impact the sorting. We can review it again when switch to the new design.